app/services/csv_generator/files.rb

```diff
@@ -19,7 +19,7 @@ def scope
           [
             doc.id,
             topic.id,
-            doc.filename,
+            topic.fullname_for_document(doc),
```

Comment:
Will this work with our imported documents? They are in Azure with their nonstandard filenames, which they should retain. The filename we have stored for those documents in SkillRX is correct and is what needs to be in the CSV.

Reply:
Good question. Let's clarify couple of thing in that other PR first